src/qtfirebaseadmob.cpp

```diff
@@ -888,6 +888,432 @@ void QtFirebaseAdMobBanner::moveTo(int position)
 
 }
 
+
+/*
+ * AdMobBanner
+ *
+ */
+QtFirebaseAdMobNativeAd::QtFirebaseAdMobNativeAd(QObject *parent) : QObject(parent)
```

Comment:
Should be probably QtFirebaseAdMobNativeExpressAd to minimize the chances of future confusion